### ISSUE TITLE
DKMv3 and DKMv2

### ISSUE THREAD

**Issue by sjtuztq** [Parskatt/DKM#41]
What is the difference between DKMv3 and DKMv2?

**Comment by Parskatt** [comment]
Quite a few changes. Main ones on the top of my head are,

1. Changes to representation of refiner input, in particular adding local correlation features.
2. Inverse KDE match sampling for better coverage
3. Higher resolution (if I remember correctly v2 used lower res)

There are probably more minor implementation details that may have given further boost.

There were no major changes to the training itself.

Hope that helps :)

**Comment by qwerty200696** [comment]
Is there any documentation available outlining the changes from v1 to v2 and then to v3?

**Comment by Parskatt** [comment]
No, you would have to basically look at changes in the source code. The initial uploaded version might even be v2, I don't remember.

**Comment by qwerty200696** [comment]
您好，我已收到您的邮件。稍后即会查阅、回复。
                                                         ——WLJ
